pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -81,6 +90,11 @@ func (c *PeriodicBackupController) sync(ctx context.Context, _ factory.SyncConte
 	}
 
 	for _, item := range backups.Items {
+		if item.Name == "default" {
+			specClone := item
+			c.backupVarGetter.SetBackupSpec(&specClone.Spec.EtcdBackupSpec)
+		}
```

Comment:
```suggestion
                  continue
		}
```
we don't want to reconcile it

Reply:
i thought i added it, my bad :'(